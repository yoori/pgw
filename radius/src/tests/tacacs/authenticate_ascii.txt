#
#	ARGV: -t ascii -u tapioca authenticate -p queijo
#
