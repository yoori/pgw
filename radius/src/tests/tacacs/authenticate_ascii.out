status: PASS
data: b'Authentication-Data'
server_msg: b'ASCII authentication accepted'
